attack.go

```diff
@@ -201,7 +201,7 @@ func attack(opts *attackOpts) (err error) {
 	res := atk.Attack(tr, opts.rate, opts.duration, opts.name)
 	enc := vegeta.NewEncoder(out)
 	sig := make(chan os.Signal, 1)
-	signal.Notify(sig, os.Interrupt)
+	signal.Notify(sig, os.Interrupt, syscall.SIGTERM)
```

Comment:
This change seems unrelated. Accidental?

Reply:
I think SIGTERM is also commonly used, so I add it. Can we add it in this pr? If not, I will remove it.

Reply:
It's fine.